code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -1534,7 +1534,7 @@
 			if (src.bleeding < 0) //INVERSE BLOOD LOSS was a fun but ultimately easily fixed bug
 				src.bleeding = 0
 
-		else if (!src.bleeding && src.get_surgery_status())
+		else if (src.bleeding && src.get_surgery_status())
```

Comment:
https://github.com/Tarmunora/goonstation/blob/5b9d352607066f82aa17d969407f63f6affe2928/code/mob/living/carbon/human/procs/Life.dm#L1518

This `if` statement means the added `else if` is never going to run; if `src.bleeding` is true the first one will run, and if it's not true this `else if` will not be true.